editor/src/components/filebrowser/fileitem.tsx

```diff
@@ -231,8 +230,7 @@ class FileBrowserItemInner extends React.PureComponent<
         isRenaming: props.renamingTarget === props.path,
       }
     }
-    // todo this is not very safe. should it be not allowed to use . in folder names?
-    const pathParts = dropLeadingSlash(props.path).split('/')
+    const pathParts = props.path.split('/').filter((s) => s !== '')
```

Comment:
hmm this is interesting. so if I name a folder `something/like/this` it becomes three nested folders?

Reply:
is that in line with how folders work on "unix" ?

Reply:
I think you can escape them with a backslash...

Reply:
![Homer Disappearing](https://media.giphy.com/media/q8C0Ljmy4F6Ss/giphy.gif)

Reply:
I've just tested with `mkdir` and inside my file browser (Dolphin) and it doesn't seem to be possible to make non-nested directories with forward slashes (even when escaped)

Reply:
https://stackoverflow.com/a/9847573